codalab/apps/web/static/less/worksheets.less

```diff
@@ -54,6 +54,9 @@
          top:-7px;
       }
    }
+   > .container {
+     width:auto;
```

Comment:
This pushes all worksheets content to the left side. Off sets the page and makes long form text spread across the entire page making it very hard to read.  
![left](https://cloud.githubusercontent.com/assets/123099/7400728/df599dea-ee73-11e4-8770-4134785e6086.png)

Why change this?